devimint/src/tests.rs

```diff
@@ -179,7 +179,11 @@ pub async fn latency_tests(
             let mut ln_sends = Vec::with_capacity(iterations);
             for i in 0..iterations {
                 let invoice = cln
-                    .invoice(1_000_000, format!("Description{i}"), format!("Label{i}"))
+                    .invoice(
+                        1_000_000,
+                        format!("Description{i}"),
+                        format!("Label{}", rand::random::<u64>()),
```

Comment:
We need unique labels for cln otherwise we get the following error

```
 Error: Error code 900: Duplicate label 'Label0'
```

@dpc this is the mystery discussed in https://discord.com/channels/990354215060795454/1314093509640192000/1314323211051073566